src/Fabulous.XamarinForms/Views/Collections/ListView.fs

```diff
@@ -69,6 +132,67 @@ module ListViewBuilders =
 
 [<Extension>]
 type ListViewModifiers =
+
+    [<Extension>]
+    static member inline header<'msg, 'marker, 'contentMarker when 'marker :> IListView and 'contentMarker :> IView>
+        (
+            this: WidgetBuilder<'msg, 'marker>,
+            content: WidgetBuilder<'msg, 'contentMarker>
+        ) =
+        this.AddWidget(ListView.Header.WithValue(content.Compile()))
+
+    [<Extension>]
+    static member inline footer<'msg, 'marker, 'contentMarker when 'marker :> IListView and 'contentMarker :> IView>
+        (
+            this: WidgetBuilder<'msg, 'marker>,
+            content: WidgetBuilder<'msg, 'contentMarker>
+        ) =
+        this.AddWidget(ListView.Footer.WithValue(content.Compile()))
+
+    [<Extension>]
+    static member inline hasUnevenRows(this: WidgetBuilder<'msg, #IListView>, value: bool) =
+        this.AddScalar(ListView.HasUnevenRows.WithValue(value))
+
+    [<Extension>]
+    static member inline horizontalScrollBarVisibility
+        (
+            this: WidgetBuilder<'msg, #IListView>,
+            value: ScrollBarVisibility
+        ) =
+        this.AddScalar(ListView.HorizontalScrollBarVisibility.WithValue(value))
+
+    [<Extension>]
+    static member inline verticalScrollBarVisibility
+        (
+            this: WidgetBuilder<'msg, #IListView>,
+            value: ScrollBarVisibility
+        ) =
+        this.AddScalar(ListView.VerticalScrollBarVisibility.WithValue(value))
+
+    [<Extension>]
+    static member inline isGroupingEnabled(this: WidgetBuilder<'msg, #IListView>, value: bool) =
```

Comment:
This one is handled directly by `View.ListView` / `View.GroupedListView`. We should not expose it